main.go

```diff
@@ -112,7 +112,9 @@ func mainE(ctx context.Context) error {
 	daemonCommand.PersistentFlags().String(f.Service.Prometheus.BaseDomain, "", "Base domain to create Prometheus Ingress resources under.")
 	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Address, "", "Address to access Prometheus UI.")
 	daemonCommand.PersistentFlags().Bool(f.Service.Prometheus.Storage.CreatePVC, false, "Should the operator create a PVC for storage.")
-	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Storage.Size, "20Gi", "Storage size for prometheus.")
+	daemonCommand.PersistentFlags().String(f.Service.Prometheus.Storage.Size, "50Gi", "Storage size for prometheus.")
```

Comment:
Should this not be 100Gi and the retention size would be 90 as well?

Reply:
You're probably right, but I always got 100Gi during my tests.

Reply:
That's because we set this value in the helm chart.

Reply:
done